### DISCUSSION TITLE
How does SHAP work for seq2seq model?

### DISCUSSION THREAD

**Discussion by ssiu** [shap/shap#2373] [Q&A]
Suppose I have a seq2seq model s with input-output s(x_1, x_2, x_3) = y_1, y_2, y_3 and we want to explain y_1, how would I go about computing equation 4 of the paper? Specifically, what is the definition of f in this case?

**Comment by jainraj** [comment]
If you see closely, this is a similar issue in categorical output models. As in that condition, here too, you can consider whatever part of the network is used to output "y_1". Hope this helps.
